src/main/java/ro/societateahermes/backendservice/entities/Event.java

```diff
@@ -21,6 +21,8 @@
     private String eventEstimatedTime;
     private LocalDateTime eventEndDate;
     private String eventLink;
+    private String imagePath;
```

Comment:
Ok, aici e o confuzie din task in sine. In ultima lui linie scrie de event, dar de fapt va trebuie o entitate separata numita post, iar entitatea asta are are nevoie de imagine, descriere si link catre original, nu eventul. puteti lasa asta aici ca fiecare event are un logo si va fi buna pentru aia :))

Reply:
Am gresit noi intelegerea cerintei initial, dar am refacut intre timp branchul din cauza unei erori. O sa redau un publish maine cel tarziu.